Mathlib/Algebra/MvPolynomial/CommRing.lean

```diff
@@ -92,6 +92,17 @@ theorem degrees_sub [DecidableEq σ] (p q : MvPolynomial σ R) :
 
 end Degrees
 
+section Degrees
+
+theorem degreeOf_neg (i : σ) (p : MvPolynomial σ R) : degreeOf i (-p) = degreeOf i p := by
```

Comment:
Can this be simp?

Reply:
Good catch, just added (also for `degrees_neg`).